src/attributes.md

```diff
@@ -357,6 +357,35 @@ pub mod m3 {
 }
 ```
 
+#### Tool lint attributes (`#[allow(clippy::foo)]`)
```

Comment:
One nit before I merge: Everything in the parenthesis should not be in the header.